lib/hypervisors.py

```diff
@@ -505,7 +521,7 @@ def test_stage(self, builder: Builder):
         stdout, _ = ssh.safe_execute(
             f'cd {self.cloud_images_path}/ && '
             f'cp {self.cloud_images_path}/tests/vagrant/Vagrantfile . '
-            f'&& vagrant box add --name almalinux-8-test *.box && vagrant up'
+            f'&& vagrant box add --name almalinux-{self.os_major_ver}-test *.box && vagrant up'
```

Comment:
We can inject the `OS_MAJOR_VER` here to be used by the test [Vagrantfile](https://github.com/AlmaLinux/cloud-images/blob/master/tests/vagrant/Vagrantfile).

```python
f'&& vagrant box add --name almalinux-{self.os_major_ver}-test *.box && OS_MAJOR_VER={self.os_major_ver vagrant up'
```

```ruby
OS_MAJOR_VER = ENV['OS_MAJOR_VER']

Vagrant.configure("2") do |config|
  config.vm.box = "almalinux/#{OS_MAJOR_VER}"
end
```